src/sub/tests/unit/test_vendor.py

```diff
@@ -1,114 +1,441 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
+import os
+import json
+from unittest import TestCase
+from mock import patch
 
-import pytest
-import stripe
-
-from stripe.error import APIError, APIConnectionError
+from stripe.error import APIError, APIConnectionError, InvalidRequestError
+from stripe.util import convert_to_stripe_object
 
 from sub.shared import vendor, utils
+from shared.exceptions import EntityNotFoundError
 
 
-def disable_base():
-    stripe.api_base = "http://example.com"
+class TestStripeCustomerCalls(TestCase):
+    def setUp(self):
+        with open(
+            os.path.join(os.path.dirname(__file__), "fixtures/stripe_cust_test1.json")
```

Comment:
same thing, calculcate dirname and avoid ugly black wrapping

Reply:
done